website/src/schema-markup/sdk-reference/cpp.json

```
@@ -0,0 +1,60 @@
+{
+  "@context": "https://schema.org/", 
+  "@type": "HowTo", 
+  "name": "How to use feature flags in C++?",
+  "description": "How to use feature flags in C++ using Configcat Feature Flags.",
```

```suggestion
  "description": "How to use feature flags in C++ using ConfigCat Feature Flags.",
```
Is the original wrong as well?

---

Hi @sigewuzhere. Nope, this is a typo on my end. Many thanks for catching that 💯